scripts/dpop-verify.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+#
+# Copyright 2023 The Sigstore Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Print all commands and stop on errors
+set -o errexit
+set -o xtrace
+set -u
+
+BRANCH=$(git rev-parse --abbrev-ref HEAD)
+FORK_POINT=$(git merge-base --fork-point origin/main "${BRANCH}")
+REPO=./repository
+DELEGATION=$1
+SIG_FILE="${REPO}"/staged/"${FORK_POINT}".sig
```

Comment:
should this be 
```suggestion
SIG_FILE="${REPO}"/staged/"${DELEGATION}".sig
```

Reply:
It doesn't really matter, the important part is that the PR is prepared correctly so the values matches. I think we can leave it as is now for testing, as the other scripts prepares the PR via the fork point.

But with the issue I linked on multiple keys (#707) this should be updated to contain the key id so we can match each signature to a key and iterate through the signatures and verify each key.